ani-cli

```diff
@@ -128,6 +135,27 @@ get_links() {
     [ -z "$ANI_CLI_NON_INTERACTIVE" ] && printf "\033[1;32m%s\033[0m Links Fetched\n" "$provider_name" 1>&2
 }
 
+download_thumbnails() {
+    [ -d "$HOME/.cache/ani-cli/images" ] || mkdir -p "$HOME/.cache/ani-cli/images"
+    printf "\33[2K\r\033[1;34mDownloading thumbnails...\033\n[0m"
```

Comment:
I suppose we should check for differing `XDG_CACHE_HOME`.
I'm also unsure if a persistent cache is warranted.

In the case of episodal thumbnails, a good case can be made for rewatchers.
In the case of search, thumbnails are disposable and belong somewhere around `/tmp`